hack/lib/test.sh

```diff
@@ -154,3 +154,20 @@ hyper::test::execvm() {
   test $res == aaa
   sudo hyperctl rm $id
 }
+
+# regression test for #542 and #537
+hyper::test::remove_container_with_volume() {
+  mkdir -p  ${HYPER_TEMP}/tmp
+  sed -e "s|TMPDIR|${HYPER_TEMP}/tmp|" ${HYPER_ROOT}/hack/pods/single-volume.pod > ${HYPER_TEMP}/single-volume.pod
+  echo "run pod with volume"
+  pod_id=$(sudo hyperctl run -d -p ${HYPER_TEMP}/single-volume.pod | sed -ne "s/POD id is \(.*\)/\1/p")
+  echo "test pod ID is $pod_id"
+  sudo hyperctl stop $pod_id
+  echo "stop pod $pod_id"
+  cid=$(sudo hyperctl list container -p $pod_id | sed -n '2p' | awk '{print $1}')
```

Comment:
I think you don't need to list the container, and could remove the container by name.